config/constants/status.js

```diff
@@ -0,0 +1,12 @@
+const status = {
+  status401: 401,
```

Comment:
`NON_AUTH`
`SUCCESS`
`SERVER_ERROR` - NEVER SEND THIS ERROR

messages are different const